src/apps_plugin.c

```diff
@@ -245,6 +246,9 @@ long get_system_cpus(void) {
 
 long get_system_pid_max(void) {
 	procfile *ff = NULL;
+
+	// FIXME: Is this a typical maximum system pid count, or just for Linux?
+	//        Is it defined on POSIX.1 ABI X/Open and/or BSD specs?
 	long mpid = 32768;
```

Comment:
Guys, the software needs a default value. 4 lines below this it reads `/proc/sys/kernel/pid_max`. 32768 is the default, if that file is not there.